### PR TITLE
Home page header & navbar

### PR DESCRIPTION
**Pull request by NadiaJalalTamimi**

### CODE REVIEW

**Review by ahmedhamouda2 — COMMENTED** [review]
Good Work  :100:

File: home-page/style.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        display: flex;
+        margin: 20px;
+        justify-content: center;
+     }
+     .activited{
+         display: block;
+     }
+
+
+}
```
Comment (Mu7ammadAbed) [inline home-page/style.css]: Remove unnecessary lines and add a new line at the end of the file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="home-page/style.css">
+    <title>News page</title>
+    <script type="text/javascript" src ="index.js" defer></script>
```
Comment (Mu7ammadAbed) [inline index.html]: You should import the js scripts at the end of the body tag

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

**Review by NourAdawi — CHANGES_REQUESTED** [review]
please add a class name to anchor tags

File: home-page/style.css (hunk 1 above)
Comment (NadiaJalalTamimi) [reply home-page/style.css]: Done

**Review by NourAdawi — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (NadiaJalalTamimi) [reply index.html]: done

**Review by Mu7ammadAbed — APPROVED** [review]

**Review by ahmedhamouda2 — APPROVED** [review]

**Pull request merged**